azurerm/resource_arm_container_group_test.go

```diff
@@ -605,7 +645,29 @@ resource "azurerm_container_group" "test" {
     secure_environment_variables = {
       "secureFoo"  = "secureBar"
       "secureFoo1" = "secureBar1"
-    }
+		}
```

Comment:
terraform configs should use spaces not tabs, as such could we revert this?
```suggestion
    }
```